### PR TITLE
Port Generate From Usage tests

### PR DESCRIPTION
**Pull request by dpoeschl**
Original tests at https://github.com/dotnet/roslyn-internal/blob/master/Closed/Hosting/RoslynTaoActions/IntegrationTests/BasicGenerateFromUsage.xml / https://github.com/dotnet/roslyn-internal/blob/master/Closed/Hosting/RoslynTaoActions/IntegrationTests/CSharpGenerateFromUsage.xml

### CODE REVIEW

**Comment by dpoeschl** [comment]
Tagging @dotnet/roslyn-ide for review

**Comment by dpoeschl** [comment]
retest windows_debug_vs-integration_prtest please

File: src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpGenerateFromUsage.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    public class CSharpGenerateFromUsage : AbstractEditorTest
+    {
+        protected override string LanguageName => LanguageNames.CSharp;
+
+        public CSharpGenerateFromUsage(VisualStudioInstanceFactory instanceFactory)
+            : base(instanceFactory, nameof(CSharpGenerateFromUsage))
+        {
+        }
+
+        [Fact]
```
Comment (rchande) [inline src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpGenerateFromUsage.cs]: Feature attribute

File: src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicGenerateFromUsage.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    public class BasicGenerateFromUsage : AbstractEditorTest
+    {
+        protected override string LanguageName => LanguageNames.VisualBasic;
+
+        public BasicGenerateFromUsage(VisualStudioInstanceFactory instanceFactory)
+            : base(instanceFactory, nameof(BasicGenerateFromUsage))
+        {
+        }
+
+        [Fact]
```
Comment (rchande) [inline src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicGenerateFromUsage.cs]: Feature attribute

**Comment by dpoeschl** [comment]
@Pilchie I added a "Generate class in new file" test.

**Pull request merged**